### ISSUE TITLE
Verifying a signed message

### ISSUE THREAD

**Issue by Fastidious** [henryboldi/felony#88]
#### What's this issue about?
Verifying a self signed message wrong strings.

#### Expected behavior
Show the name/email of the person that signed the message.

#### Actual behavior
Show undefined <undefined>.

#### Steps to reproduce
Create key, sign a message, verify it.

#### Environment
macOS Sierra.

![screen shot 2016-10-24 at 9 55 53 am](https://cloud.githubusercontent.com/assets/8352292/19648417/0ffff062-99d0-11e6-82bb-ac3355b1286d.png)

**Comment by frankcash** [comment]
was the user part of your "Friends" list?

**Comment by Fastidious** [comment]
I was the user. I signed a message, and I verified. That screenshot is the verification part.

**Comment by frankcash** [comment]
Yes , we need to account for this. I know if you add yourself it will work.
If you'd like to submit a PR to fix this that would be great!

**Comment by Fastidious** [comment]
@frankcash if I knew, I would have submitted one to begin with. :-)

**Comment by ldabiralai** [comment]
I just tried to reproduce this following the steps above and it seemed to work so we should close this, unless someone else can verify it as not working

**Comment by Fastidious** [comment]
@ldabiralai "it seemed to work?" This either works, or doesn't. This is still an issue. When someone produces a PR to account for [this](https://github.com/henryboldi/felony/issues/88#issuecomment-256388744), that someone will close the issue.

**Comment by ldabiralai** [comment]
@Fastidious I said it seemed to work because I'm not that well versed in the expected behaviour.

I followed the steps you described above and received this message:

![image](https://user-images.githubusercontent.com/1711610/31315764-ee87abdc-ac16-11e7-9d2b-377d862220d3.png)

I added my own public key to my contact list and then received this:

![image](https://user-images.githubusercontent.com/1711610/31315773-1b7faac2-ac17-11e7-852f-68ef6dbbc1c1.png)

To me, both of these behaviours look correct..

**Comment by Fastidious** [comment]
@ldabiralai I should not need to be on the contact list to sign a message. Why will I need to be on my contact list? That is exactly what needs to be accounted for, which @frankcash noted on his comment.

**Comment by meMuszr** [comment]
Working on this if that's okay ~

**Comment by meMuszr** [comment]
[npm-debug.log](https://github.com/henryboldi/felony/files/1385135/npm-debug.log) 
Unfortunately couldn't even get this to run.. :\
